ng-pkgs/activation_service/sals/activation_sal.py

```diff
@@ -0,0 +1,15 @@
+from jumpscale.loader import j
```

Comment:
Do we need to have this in a seperate file instead of in the package? Now the sys.path has to modified instead of simply calling the methods on the package

Reply:
We moved it as the package.py  to not include functionality in it not related to the locations and starting the package. To import the package.py directly from the actor I believe we will also need to modify the sys.path so this was an abstraction to just seperate functionalities .